project_euler/problem_13/sol1.py

```diff
@@ -1,30 +1,30 @@
 """
+Problem 13: https://projecteuler.net/problem=13
+
 Problem Statement:
 Work out the first ten digits of the sum of the following one-hundred 50-digit
 numbers.
 """
+import os
 
 
-def solution(array):
-    """Returns the first ten digits of the sum of the array elements.
+def solution():
+    """
+    Returns the first ten digits of the sum of the array elements
+    from the file num.txt
 
-    >>> import os
-    >>> sum = 0
-    >>> array = []
-    >>> with open(os.path.dirname(__file__) + "/num.txt","r") as f:
-    ...     for line in f:
-    ...         array.append(int(line))
-    ...
-    >>> solution(array)
+    >>> solution()
     '5537376230'
     """
+    array = []
```

Comment:
with open(os.path.join(os.path.dirname(__file__), "num.txt"), "r") as in_file:
         return str(sum([int(line) for line in in_file][:10]))